### PR TITLE
Release 1.15.0

### PR DESCRIPTION
**Pull request by vrubezhny**

### CODE REVIEW

**Review by msivasubramaniaan — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 
+## 1.15.0 (July 16, 2024)
+
+ * [#4283](https://github.com/redhat-developer/vscode-openshift-tools/pull/4283) Added validation for container image to be deployed from an image URL
+ * [#4255](https://github.com/redhat-developer/vscode-openshift-tools/issues/4255) Remove outdated method from the extension activation
+ * [#4247](https://github.com/redhat-developer/vscode-openshift-tools/pull/4247) Remove dependency to 'make-fetch-happen' NPM module
+ * [#4233](https://github.com/redhat-developer/vscode-openshift-tools/issues/4233) Cluster Webview: sandbox detection status never ends
+ * [#4183](https://github.com/redhat-developer/vscode-openshift-tools/issues/4183) ODO CLI binary stopped working after Devfile schema update to v.2.2.2
+ * [#4172](https://github.com/redhat-developer/vscode-openshift-tools/issues/4172) There is no any error/warning notification when Helm Chart cannot be installed
+ * [#4166](https://github.com/redhat-developer/vscode-openshift-tools/pull/4166) Trim token value from Clipboard when loggin in to DevSandbox
```
Comment (rgrunber) [inline CHANGELOG.md:11]: loggin -> logging

Comment (vrubezhny) [reply CHANGELOG.md:11]: ACK

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 1.15.0 (July 16, 2024)
+
+ * [#4283](https://github.com/redhat-developer/vscode-openshift-tools/pull/4283) Added validation for container image to be deployed from an image URL
+ * [#4255](https://github.com/redhat-developer/vscode-openshift-tools/issues/4255) Remove outdated method from the extension activation
+ * [#4247](https://github.com/redhat-developer/vscode-openshift-tools/pull/4247) Remove dependency to 'make-fetch-happen' NPM module
+ * [#4233](https://github.com/redhat-developer/vscode-openshift-tools/issues/4233) Cluster Webview: sandbox detection status never ends
+ * [#4183](https://github.com/redhat-developer/vscode-openshift-tools/issues/4183) ODO CLI binary stopped working after Devfile schema update to v.2.2.2
+ * [#4172](https://github.com/redhat-developer/vscode-openshift-tools/issues/4172) There is no any error/warning notification when Helm Chart cannot be installed
```
Comment (rgrunber) [inline CHANGELOG.md:10]: Here, I would just say : "There is no error/warning notification when Helm Chart cannot be installed"

**Comment by rgrunber** [comment]
Should we mention any of :

https://github.com/redhat-developer/vscode-openshift-tools/pull/4170
https://github.com/redhat-developer/vscode-openshift-tools/pull/4256

**Comment by vrubezhny** [comment]
> Should we mention any of :
> 
> #4170 #4256

I don't think so, The "dedupe" is a small issue which is a part of https://github.com/redhat-developer/vscode-openshift-tools/issues/3823 while the "extension is slow on activate" is not really finished yet and is a part of the issue that still needs more work to be done in order to fix it.

**Review by rgrunber — DISMISSED** [review]

**Comment by datho7561** [comment]
What about d3ae3244 ?

**Review by datho7561 — DISMISSED** [review]
Other than *maybe* that one commit I mentioned, this seems good to me

**Pull request merged**